src/uhttp.c

```diff
@@ -320,7 +321,36 @@ static int write_text_line(HTTP_CLIENT_HANDLE_DATA* http_data, const char* text_
         result = 0;
         if (http_data->trace_on)
         {
-            LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+            char* authStart = strstr(text_line, "Authorization:");
+            char* proxyAuthStart = strstr(text_line, "Proxy-Authorization");
+            if (authStart == NULL && proxyAuthStart == NULL)
+            {
+                LOG(AZ_LOG_TRACE, LOG_LINE, "%s", text_line);
+            }
+            else if (authStart != NULL && proxyAuthStart == NULL)
+            {
+                char* authEol = strstr(authStart, "\x0D\x0A");
+                if (authEol != NULL)
```

Comment:
If this is null, should we still print up to the authStart or something like that, or does this mean that the `text_line`  has something wrong with it so it's not worth printing?

Reply:
(same question for line 341)

Reply:
I'm fine not printing it. mal-formed HTML, should not happen, this is only trace code

Reply:
I'm only printing for well formed HTML.